### ISSUE TITLE
Cannot add Google Music

### ISSUE THREAD

**Issue by derTobsch** [pascalopitz/unoffical-sonos-controller-for-linux#23]
Failure in MusicServiceClient.js in line 58
```
MusicServiceClient.js:58 Uncaught (in promise) Error: Invalid link code in request.(…)(anonymous function) @ MusicServiceClient.js:58self.callback @ request.js:186emitTwo @ events.js:106emit @ events.js:191(anonymous function) @ request.js:1081emitOne @ events.js:96emit @ events.js:188(anonymous function) @ request.js:1001g @ events.js:286emitNone @ events.js:91emit @ events.js:185endReadableNT @ _stream_readable.js:974_combinedTickCallback @ internal/process/next_tick.js:74_tickCallback @ internal/process/next_tick.js:98
```

I just clicked on NEXT on the first screen and then there is a google-faq link from sonos

**Comment by derTobsch** [comment]
ok this is a duplicate of #19 :)

**Comment by pascalopitz** [comment]
This might be fixed now as of 0.0.14
